config/database.yml

```diff
@@ -8,6 +8,8 @@ default: &default
   adapter: postgresql
   pool: 5
   timeout: 5000
+  username: playthisforus
```

Comment:
@mstojcevich We may want to use environment variables for `username`/`password` rather than hardcoding, thoughts?

Eg: `username: ENV['DB_USERNAME']`

Then add `.env` with `DB_USERNAME=playthisforus` and run with `foreman start` (foreman will load variables in `.env`)